modules/persistent-storage-local-install.adoc

```diff
@@ -60,25 +57,25 @@ Once finished, the Local Storage Operator will be listed in the *Installed Opera
 . Install the Local Storage Operator from the CLI.
```

Comment:
In the "From the UI" section above (line 51/48), should the user "Select *openshift-local-storage* from the drop-down menu." ? Or is this still *local-storage*?